### PR TITLE
PAYARA-3308 Upgrade PrototypeJS version used in the Admin Console

### PR DESCRIPTION
**Pull request by MattGill98**
Depends on https://github.com/payara/Payara_PatchedProjects/pull/207.

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by MattGill98** [comment]
jenkins test please

**Comment by MattGill98** [comment]
jenkins test please

File: appserver/admingui/war/src/main/webapp/WEB-INF/sun-web.xml [hunk 1]
```diff
@@ -61,6 +61,6 @@
 	<parameter-encoding default-charset="UTF-8" />
  </locale-charset-info>
 
-<class-loader delegate="true" extra-class-path="WEB-INF/extra/webui-jsf-suntheme-4.0.2.10.payara-p15.jar:WEB-INF/extra/dojo-ajax-nodemo-0.4.4.jar:WEB-INF/extra/webui-jsf-4.0.2.10.payara-p15.jar:WEB-INF/extra/commons-fileupload-1.3.3.jar:WEB-INF/extra/commons-io-2.6.jar:WEB-INF/extra/json-1.0.jar:WEB-INF/extra/prototype-1.5.0.jar" />
+<class-loader delegate="true" extra-class-path="WEB-INF/extra/webui-jsf-suntheme-4.0.2.10.payara-p15.jar:WEB-INF/extra/dojo-ajax-nodemo-0.4.4.jar:WEB-INF/extra/webui-jsf-4.0.2.10.payara-p15.jar:WEB-INF/extra/commons-fileupload-1.3.3.jar:WEB-INF/extra/commons-io-2.6.jar:WEB-INF/extra/json-1.0.jar:WEB-INF/extra/prototype-1.7.3.jar" />
```
Comment (Pandrex247) [inline appserver/admingui/war/src/main/webapp/WEB-INF/sun-web.xml]: Has the suntheme not also been upgraded to p16?

Comment (MattGill98) [reply appserver/admingui/war/src/main/webapp/WEB-INF/sun-web.xml]: No, turns out that a woodstock update isn't required. I've reverted the patched projects commit as well.

**Pull request merged**